tests/extmod_hardware/machine_pwm.py

```diff
@@ -0,0 +1,108 @@
+# Test machine.PWM, frequncy and duty cycle (using machine.time_pulse_us).
+#
+# IMPORTANT: This test requires hardware connections: the PWM-output and pulse-input
+# pins must be wired together.
```

Comment:
```suggestion
# pins must be wired together (see variables "pwm_pin" and "pulse_pin").
```

This is evident after you read and understanding the code, but if someone is running the whole test suite and doesn't overly care about PWM then it's all the information they need.

Reply:
Done.